### DISCUSSION TITLE
Unable to receive admin's magic link email after successfuly deployment

### DISCUSSION THREAD

**Discussion by firew0rks** [mozilla/hubs#2968] [General Questions & Discussion] [answered]
Hey!

I've tried deploying Hubs Cloud Personal on AWS  and everything deployed successfully. But when I try to sign in with the admin email that I specified during stack creation, I don't receive the magic link email even though I see the message on the client as "Email sent to <admin-email>@gmail.com! ...". I've verified the admin email by clicking on the link sent by AWS. In my SES console, it shows both the domain and email address as verified on us-east-1. The admin email specified contains no caps. Am I missing anything else, or is there a place where I can check why the emails aren't sending?

Thanks

**Comment by robinkwilson** [comment, marked as answer]
Thanks for checking all of those places! Very interesting.

In the SES console, do you see the email outbound count go up?
Double check no caps in the SES console + in CloudFormation stack parameters? -- sanity check just for me

If you don't have any luck, please email hubs@mozilla.com for use to look into this further.

For people searching this issue, here's instructions of where to check for your admin email address:
https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#then-what-is-my-hub-stacks-admin-email-address
